test/model/algebra/vector/vector.cpp

```diff
@@ -3,5 +3,6 @@
 #include <catch2/catch.hpp>
 #include <vector>
 
+// TODO
```

Comment:
I would've put the TODO inside the empty test, but whatever ^^

Reply:
removed

Reply:
no?

Reply:
yes